### PR TITLE
Add "tweet" intros to head lice and warts

### PR DESCRIPTION
**Pull request by Demotive**

### CODE REVIEW

**Comment by fawkesley** [comment]
Don't merge this right now, it hasn't gone through @pacharanero

File: head-lice.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ condition: Head lice
 
 #Head lice
```
Comment (Demotive) [inline head-lice.md:7]: @pacharanero - can you check this tweet sized intro?

File: warts.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ condition: Warts
 
 #Warts
```
Comment (Demotive) [inline warts.md:7]: @pacharanero this one too?

File: head-lice.md [hunk 3]
```diff
@@ -5,6 +5,9 @@ condition: Head lice
...
 
+Treat head lice at home with lotion, spray and a special comb from the pharmacy – no need to see your GP.
```
Comment (pacharanero) [inline head-lice.md:8]: I'm happy with this

File: warts.md [hunk 4]
```diff
@@ -5,6 +5,10 @@ condition: Warts
...
 
+Warts are harmless although they can spread if scratched. They often go away on
+their own although you may be able to speed this up with certain creams.
```
Comment (pacharanero) [inline warts.md:9]: I'm happy with this

**Pull request merged**